.github/workflows/integration_tests.yml

```
@@ -0,0 +1,31 @@
+# Workflow to run integration tests
+# Checks that check_new_repo workflow works as expected
+
+name: Integration Tests
+
+on:
+  push:
+    branches-ignore:
+      - integration-test-1
+    paths:
+      - custom_python_actions/*
+      - tests/*
+      - .github/workflows/*
+  merge_group:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  integration-test-1:
+    name: Integration Test 1
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Integration Test 1
+      run: |
+        set -euo pipefail
+        bash ${GITHUB_WORKSPACE}/.github/integration_tests/nonexistent_repo.sh
```

```suggestion
        ${GITHUB_WORKSPACE}/.github/integration_tests/nonexistent_repo.sh
```
Better to directly run script that have shebang. Otherwise you may have different behavior when run from CI or locally because you use different interpreter by mistake.